library/alloc/src/vec/mod.rs

```diff
@@ -542,8 +542,8 @@ impl<T> Vec<T> {
     ///
     /// unsafe {
     ///     // Overwrite memory with 4, 5, 6
-    ///     for i in 0..len as isize {
```

Comment:
So nice that using `add` makes the `for` loop nicer like this 👍

Reply:
ikr